.pre-commit-config.yaml

```diff
@@ -44,20 +44,18 @@ repos:
       - id: yamllint
         args:
           - --config-file
-          - .yamllint.yaml
+          - .yamllint.config.yaml
```

Comment:
this is not a good idea as the default is .yamllint.yaml

see https://yamllint.readthedocs.io/en/stable/configuration.html